include/openssl/core_names.h

```diff
@@ -191,6 +194,10 @@ extern "C" {
 #define OSSL_SIGNATURE_PARAM_DIGEST         "digest"
 #define OSSL_SIGNATURE_PARAM_DIGEST_SIZE    "digest-size"
 
+/* Keygen callback parameters, from provider to libcrypto */
+#define OSSL_GEN_PARAM_POTENTIAL            "potential" /* integer */
```

Comment:
What the heck is this :)

Reply:
Not my fault is what it is :-p

The [manual for BN_generate_prime_ex](https://www.openssl.org/docs/man1.1.1/man3/BN_generate_prime_ex.html) has a little more to say on the matter:

> If **cb** is not NULL, it is used as follows:
> 
> -    **BN_GENCB_call(cb, 0, i)** is called after generating the i-th potential prime number.
> -    While the number is being tested for primality, **BN_GENCB_call(cb, 1, j)** is called as described below.
> -    When a prime has been found, **BN_GENCB_call(cb, 2, i)** is called.
> -    The callers of BN_generate_prime_ex() may call **BN_GENCB_call(cb, i, j)** with other values as described in their respective man pages; see "SEE ALSO".

So "potential" is that 'i-th potential prime number'

Better suggestions welcome!  The meaning of these callback args are baddly documented, so I'm frankly grasping at straws.